ws-nextjs-app/pages/[service]/send/[id]/styles.ts

```diff
@@ -2,29 +2,56 @@ import { Theme, css } from '@emotion/react';
 import pixelsToRem from '../../../../../src/app/utilities/pixelsToRem';
 
 export default {
+  background: () =>
+    css({
+      position: 'relative',
+      background: 'rgb(100, 5, 5)',
+    }),
+  gradient: () =>
```

Comment:
Do we need something like `zIndex: -1` here? Otherwise this will probably sit on top of the rest of the content and make the form unclickable.

Reply:
no longer needed now we're using css gradient 😊